glom/test/test_error.py

```diff
@@ -230,6 +230,27 @@ def __repr__(self):
     assert '(len=' not in actual  # no length on a single object
 
 
+def test_branching_stack():
+    assert _make_stack(Match(Switch(
+            [(1, 1), ('a', 'a'), (T.a, T.a)]))) == """\
+Traceback (most recent call last):
+  File "test_error.py", line ___, in _make_stack
+    glom(target, spec)
+  File "core.py", line ___, in glom
+    raise err
+glom.matching.MatchError: error raised while processing, details below.
+ Target-spec trace (most recent last):
+ - Target: [None]
+ - Spec: Match(Switch([(1, 1), ('a', 'a'), (T.a, T.a)]))
+ - Spec: Switch([(1, 1), ('a', 'a'), (T.a, T.a)])
+   Failed Branch ([None] does not match 1):
+   - Spec: 1
+   Failed Branch ([None] does not match 'a'):
+   - Spec: 'a'
```

Comment:
so I see `'a'` duplicated here, when would it not be? I'm thinking maybe we change the bullet notation (currently always `-`) for branches to be kind of "connected", more tersely showing that things are done in parallel maybe? Like, actual tree notation?

Reply:
the first 'a' is coming from `str(MatchError)`, the second is coming from `repr(spec)`

Reply:
are you thinking an ascii art tree?